core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/service/ReactorFlowManager.java

```diff
@@ -233,13 +233,14 @@ public FlowIdentifier triggerClusterReInstall(Long stackId) {
 
     public FlowIdentifier triggerDatalakeClusterUpgrade(Long stackId, String imageId) {
         String selector = FlowChainTriggers.DATALAKE_CLUSTER_UPGRADE_CHAIN_TRIGGER_EVENT;
-        return reactorNotifier.notify(stackId, selector, new ClusterUpgradeTriggerEvent(selector, stackId, imageId));
+        return reactorNotifier.notify(stackId, selector, new ClusterUpgradeTriggerEvent(selector, stackId, imageId, false));
```

Comment:
Here the `rollingUpgradeEnabled` is always `rollingUpgradeEnabled`.
Should it not work the same way for Data Lakes? Or is it just a preparation?
The `ClusterUpgradeTriggerEvent` expects this flag too.

Reply:
The DL part is under implementation, it will use the rollingUpgradeEnabled flag instead of the `false`